bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,236 @@
+			<channel id="channelMute" typeId="channelMute"/>
+			<channel id="channelVolume" typeId="channelVolume"/>
+			<channel id="channelInput" typeId="channelInput"/>
+			<channel id="channelSoundProgram" typeId="channelSoundProgram"/>
+			<channel id="channelSelectPreset" typeId="channelSelectPreset"/>
+			<channel id="channelSleep" typeId="channelSleep"/>
+			<channel id="channelMCServer" typeId="channelMCServer"/>
+			<channel id="channelUnlinkMCServer" typeId="channelUnlinkMCServer"/>
+			<channel id="channelRecallScene" typeId="channelRecallScene"/>
+			<channel id="channelVolumeAbs" typeId="channelVolumeAbs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone3Controls">
+		<label>Zone 3</label>
+		<channels>
+			<channel id="channelPower" typeId="channelPower"/>
+			<channel id="channelMute" typeId="channelMute"/>
+			<channel id="channelVolume" typeId="channelVolume"/>
+			<channel id="channelInput" typeId="channelInput"/>
+			<channel id="channelSoundProgram" typeId="channelSoundProgram"/>
+			<channel id="channelSelectPreset" typeId="channelSelectPreset"/>
+			<channel id="channelSleep" typeId="channelSleep"/>
+			<channel id="channelMCServer" typeId="channelMCServer"/>
+			<channel id="channelUnlinkMCServer" typeId="channelUnlinkMCServer"/>
+			<channel id="channelRecallScene" typeId="channelRecallScene"/>
+			<channel id="channelVolumeAbs" typeId="channelVolumeAbs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone4Controls">
+		<label>Zone 4</label>
+		<channels>
+			<channel id="channelPower" typeId="channelPower"/>
+			<channel id="channelMute" typeId="channelMute"/>
+			<channel id="channelVolume" typeId="channelVolume"/>
+			<channel id="channelInput" typeId="channelInput"/>
+			<channel id="channelSoundProgram" typeId="channelSoundProgram"/>
+			<channel id="channelSelectPreset" typeId="channelSelectPreset"/>
+			<channel id="channelSleep" typeId="channelSleep"/>
+			<channel id="channelMCServer" typeId="channelMCServer"/>
+			<channel id="channelUnlinkMCServer" typeId="channelUnlinkMCServer"/>
+			<channel id="channelRecallScene" typeId="channelRecallScene"/>
+			<channel id="channelVolumeAbs" typeId="channelVolumeAbs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="playerControls">
+		<label>player Controls</label>
+		<channels>
+			<channel id="channelPlayer" typeId="channelPlayer"/>
+			<channel id="channelArtist" typeId="channelArtist"/>
+			<channel id="channelTrack" typeId="channelTrack"/>
+			<channel id="channelAlbum" typeId="channelAlbum"/>
+			<channel id="channelAlbumArt" typeId="channelAlbumArt"/>
+			<channel id="channelRepeat" typeId="channelRepeat"/>
+			<channel id="channelShuffle" typeId="channelShuffle"/>
+		</channels>
+	</channel-group-type>
+
+
+
+
+	<channel-type id="channelPower">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power channel</description>
+	</channel-type>
```

Comment:
Instead of defining your own channel, you could use the system channel. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Please check all.

Reply:
Power and Mute have been updated, available via new push